Makefile

```
@@ -119,10 +119,15 @@ HTML_DST = \
   $(patsubst _extras/%.md,${DST}/%/index.html,$(sort $(wildcard _extras/*.md))) \
   ${DST}/license/index.html
 
+## * install-rmd-deps : Install R packages dependencies to build the RMarkdown lesson
+install-rmd-deps:
+	Rscript -e 'source("bin/dependencies.R"); install_dependencies(identify_dependencies())'
+
 ## * lesson-md        : convert Rmarkdown files to markdown
 lesson-md : ${RMD_DST}
 
-_episodes/%.md: _episodes_rmd/%.Rmd
+_episodes/%.md: _episodes_rmd/%.Rmd install-rmd-dependencies
+	mkdir -p _episodes
```

I believe `_episodes` exists in the default template (via `.gitkeep`). Why do we need to create it here?

In case we do, we can suppress Makefile's output with `@`
```suggestion
	@mkdir -p _episodes
```

---

It was removed at least from https://github.com/swcarpentry/r-novice-gapminder (also maybe in others).

---

I included the `@`